src/proj.rs

```diff
@@ -124,15 +124,18 @@ impl Area {
 }
 
 /// Easily get a String from the external library
-pub(crate) fn _string(raw_ptr: *const c_char) -> Result<String, ProjError> {
-    let c_str = unsafe { CStr::from_ptr(raw_ptr) };
+pub(crate) unsafe fn _string(raw_ptr: *const c_char) -> Result<String, ProjError> {
```

Comment:
Since this function takes in a pointer and dereferences it, it's possible for someone (one of us since it's `pub(crate)`) to misuse it and reference invalid memory and cause undefined behavior, so we need to mark it as `unsafe`

Reply:
The latest force push marks a couple functions in `network.rs` as `unsafe` for the same reason. They surfaced in CI because `_string` gets used within those network functions.